src/foobar/models.py

```diff
@@ -15,7 +15,11 @@
 class Account(UUIDModel, TimeStampedModel):
     user = models.ForeignKey(User, null=True, blank=True)
     name = models.CharField(null=True, blank=True, max_length=128)
-    email = models.CharField(null=True, blank=True, max_length=128)
+    email = models.EmailField(null=True, blank=True)
+
+    @property
+    def isComplete(self):
+        return bool(self.email)
```

Comment:
It is not just our convention. PEP8 does actually say that the underscore case is the preferred naming method.

> **Method Names and Instance Variables**
> Use the function naming rules: lowercase with words separated by underscores as necessary to improve readability.

Source: https://www.python.org/dev/peps/pep-0008/#id46